Cargo.toml

```diff
@@ -42,11 +42,10 @@ members = [
   "lib/k8s-test-framework",
   "lib/portpicker",
   "lib/prometheus-parser",
-  "lib/vector-api-client",
   "lib/remap-cli",
-  "lib/remap-lang",
```

Comment:
I presume these are not supposed to be removed?

Reply:
I'm also wondering about this.

Reply:
They were in the list twice. I should have remarked on that in the PR. Apologies.